android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -99,7 +98,6 @@ public void handleVisibility() {
     @Override
     public void onDestroyView() {
         super.onDestroyView();
-        OpenEventApp.getEventBus().unregister(this);
```

Comment:
There is no event bus registered for the fetching of feed event the reason being that rxJava was used to subscribe to feed observable.